docs/usage/configuration-options.md

```diff
@@ -1248,6 +1248,27 @@ This way the secret can be used in both the `org1` and `org2` organizations.
 
 For more information on how to use secrets for private packages, read [Private package support](./getting-started/private-packages.md).
 
+## env
+
+This option allows users to specify explicit environment variables and their corresponding values.
+
+<!-- prettier-ignore -->
+!!! note
```

Comment:
```suggestion
!!! warning
```

Should we bump this to `warning`? I think a note is easier to ignore than a warning type message. Reading the current docs in this PR, it looks like the bot admin _must_ do some other config first, before end-users can use the `env` config option. So we should shout a bit louder about the required steps, I think.